airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/write/BasicFunctionalityIntegrationTest.kt

```diff
@@ -458,9 +455,9 @@ abstract class BasicFunctionalityIntegrationTest(
     @Test
     open fun testNamespaces() {
         assumeTrue(verifyDataWriting)
-        fun makeStream(namespace: String?, name: String = "test_stream") =
+        fun makeStream(namespace: String?) =
             DestinationStream(
-                DestinationStream.Descriptor(namespace, name),
+                DestinationStream.Descriptor(namespace, "test_stream"),
```

Comment:
implementing my comment from https://github.com/airbytehq/airbyte/pull/52074#discussion_r1939856528

Reply:
ah derp, was this to prevent collisions between test attempts? lemme just randomize all the stream names the same way